src/unix/aix.c

```diff
@@ -857,7 +856,7 @@ int uv_fs_event_start(uv_fs_event_t* handle,
   /* Setup/Initialize all the libuv routines */
   uv__handle_start(handle);
   uv__io_init(&handle->event_watcher, uv__ahafs_event, fd);
-  handle->path = uv__strdup((const char*)&absolute_path);
+  handle->path = uv__strdup(filename);
```

Comment:
can you provide a quick overview in respect to why this is needed instead of the absolute path